src/app/twinclass/page.tsx

```diff
@@ -173,24 +309,21 @@ export default function TwinClasses() {
             return Promise.resolve();
           },
         }}
-        customizableColumns={{
-          enabled: true,
-          defaultVisibleKeys: [
-            "name",
-            "headClassId",
-            "extendsClass.id",
-            "abstractClass",
-          ],
-        }}
-      />
-
-      <TwinClassDialog
-        open={classDialogOpen}
-        onOpenChange={(newOpen) => {
-          setClassDialogOpen(newOpen);
-        }}
-        onSuccess={() => tableRef.current?.refresh()}
+        defaultVisibleColumns={[
+          colDefs.id,
+          colDefs.key,
+          colDefs.name,
+          colDefs.headClassId,
+          colDefs.extendsClassId,
+          colDefs.abstractClass,
+        ]}
+        dialogForm={twinClassesForm}
+        onCreateSubmit={handleOnCreateSubmit}
+        onUpdateSubmit={handleOnUpdateSubmit}
```

Comment:
I don't think we should have update logic as a modal in twin classes, let's remove it. It should be in twin class general and other tabs.